projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/K8SMemoryConversionUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service.deploy;
+
+public class K8SMemoryConversionUtils {
+
+  public static int getMemoryInMi(String memory, int amount) {
```

Comment:
It's minor thing but it bugs me. I would expect the value to be first and then the unit (1 KB not KB 1) 

ALso memory is misleading a bit variable name. Better if it is "memoryUnit"

```suggestion
  public static int getMemoryInMi(int amount, String memoryUnit) {
```